install/geoip.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+
+if [ ! -f 'install.sh' ]; then echo 'Where are you?'; exit 1; fi
+
+dc="docker-compose --no-ansi"
+dcr="$dc run --rm"
+
+
+install_geoip() {
+  local mmdb='geoip/GeoLite2-City.mmdb'
+  local conf='geoip/GeoIP.conf'
+  local result='Done'
+
+  echo "Setting up IP address geolocation ..."
+  if [[ ! -f "$mmdb" ]]; then
+    echo -n "Installing (empty) IP address geolocation database ... "
+    cp "$mmdb.empty" "$mmdb"
+    echo "done."
+  else
+    echo "IP address geolocation database already exists."
+  fi
+
+  if [[ ! -f "$conf" ]]; then
+    echo "IP address geolocation is not configured for updates."
+    echo "See https://develop.sentry.dev/self-hosted/geolocation/ for instructions."
```

Comment:
Just thought about this, maybe we can try something like https://unix.stackexchange.com/a/437581/108960?

Reply:
Would we try to sniff the terminal in use? Terminal.app is noticeably absent from the list of supported terminals. :-/

Reply:
Ehh, then forget it. Not worth the effort I'd say.